src/applications/vre/28-8832/wizard/pages/IneligibleNotice.jsx

```diff
@@ -8,6 +9,10 @@ import {
 const IneligibleNotice = () => {
   useEffect(() => {
     sessionStorage.setItem(WIZARD_STATUS, WIZARD_STATUS_INELIGIBLE);
+    recordEvent({
+      event: `howToWizard-notice-displayed`,
+      'reason-for-notice': 'ineligibility - not a service member or veteran',
```

Comment:
Can we include dependent in here as well? So something like `ineligibility  not a service member or veteran, or a dependent of one`

Reply:
These were set up by the analytics team, I guess they have a system for how they assign these